velox/common/base/PeriodicStatsReporter.cpp

```diff
@@ -228,4 +233,11 @@ void PeriodicStatsReporter::reportCacheStats() {
   lastCacheStats_ = cacheStats;
 }
 
+void PeriodicStatsReporter::reportSpillStats() {
+  const auto spillMemoryStats = spillMemoryPool_->stats();
+  LOG(INFO) << "Spill memory usage: " << spillMemoryPool_->toString();
```

Comment:
Oh, it is better to just report the two memory stats. I was thinking to log the spill stats from globalSpillStats()